src/layman/user/rest_user.py

```diff
@@ -52,7 +52,13 @@ def delete(username):
         x_forwarded_items=x_forwarded_items,
         actor_name=username,
     )
-    delete_user(username)
-    return jsonify({
-        'username': username
-    }), 200
+    skipped_publications = delete_user_public_publications(username)
+    if not skipped_publications:
+        delete_user(username)
+        return jsonify({
+            'username': username
+        }), 200
+
+    raise LaymanError(58, {
```

Comment:
This will leave user publication and also some publication in public workspaces deleted. I prefere to run checks before deleting anything.